velox/parse/QueryPlanner.cpp

```diff
@@ -108,15 +108,18 @@ PlanNodePtr toVeloxPlan(
     QueryContext& queryContext) {
   std::vector<std::string> names;
   std::vector<TypePtr> types;
+  std::vector<VectorPtr> children;
   for (auto i = 0; i < logicalDummyScan.types.size(); ++i) {
     names.push_back(queryContext.nextColumnName());
     types.push_back(duckdb::toVeloxType(logicalDummyScan.types[i]));
+    children.emplace_back(
+        BaseVector::createNullConstant(types.back(), 1, pool));
   }
 
   auto rowType = ROW(std::move(names), std::move(types));
 
-  std::vector<RowVectorPtr> vectors = {std::make_shared<RowVector>(
-      pool, rowType, nullptr, 1, std::vector<VectorPtr>{})};
+  std::vector<RowVectorPtr> vectors = {
+      std::make_shared<RowVector>(pool, rowType, nullptr, 1, children)};
```

Comment:
is this related to the fix? this looks like an invalid vector in general

Reply:
It was a generally invalid vector, but the tests that produce a LogicalDummyScan in duckdb don't actually use this Vector.

E.g. this happens when you SELECT from an UNNEST expression instead of a table
SELECT x, x + 5 FROM UNNEST([1, 2, 3]) as t(x)

Here we want to read a row from somewhere (the ValuesOperator), but it's children are never accessed, it's enough that the vector simply exists.

With this change the ValuesOperator is copying the Vector, so now they are accessed.  We still don't care what's in it, but we do care that it's valid.